dotfiles/install_config.py

```diff
@@ -72,56 +95,102 @@ def main() -> None:
     parser = argparse.ArgumentParser(description="Reads a TOML file and prints its contents.")
     parser.add_argument("-f", "--filename", help="The path to the TOML file", default="config.toml")
     parser.add_argument("-m", "--machine", help="The machine in which use the template from", required=False)
+    parser.add_argument("-s", "--shell", help="shell on which to install | bash or powershell", required=False, default="bash")
+    parser.add_argument("--install", help="Install the environment", action=argparse.BooleanOptionalAction, required=False, default=True)
     args = parser.parse_args()
-
+    install = args.install
     config = read_toml_file(filename=args.filename)
-    
-    # get home directory 
-    home = os.environ("HOME")
 
-    # Change bash aliases 
-    #====================
+
+    # get home directory
+    # =================
+    if not install:
+        home = Path.cwd()
+    else:
+        home = Path.home()
+
+    # Change bash aliases
+    # ====================
     alias = render_template(template_path="bash/bash_aliases", context=config)
 
     # write bash aliases to $home/.bash_aliases
-    with open(file=f"{home}/.bash_aliases", mode="a") as f:
+    with open(file=f"{home}/.bash_aliases", mode="w", encoding='utf-8') as f:
         f.write(alias)
 
     # Change bash functions
-    #======================
+    # ======================
     functions = render_template(template_path="bash/bash_functions", context=config)
 
     # write bash functions to $home/.bash_functions
-    with open(file=f"{home}/.bash_functions", mode="a") as f:
+    # print(functions)
+    with open(file=f"{home}/.bash_functions", mode="w", encoding='utf-8') as f:
         f.write(functions)
 
     # Change bash profile
-    #====================
-    profile = render_template(template_path="bash/bash_profile", context=config)
-
-    # write bash profile to $home/.bash_profile
-    with open(file=f"{home}/.bash_profile", mode="a") as f:
-        f.write(profile)
+    # ====================
+    if args.shell.lower() == 'bash':
+        profile = render_template(template_path="bash/bash_profile", context=config)
+
+        machine_templates = {
+            "hera": "hera",
+            "orion": "orion",
+            "hercules": "hercules",
+            "mac": "mac",
+            "wcoss": "wcoss",
+            "niagara": "niagara",
+            "gaea": "gaea"
+        }
+        machine_profile = render_template(template_path=f"bash/machines/{machine_templates.get(args.machine, '')}", context=config)
```

Comment:
The error message would be more clear in this case?
```suggestion
        machine_profile = render_template(template_path=f"bash/machines/{machine_templates[args.machine]}", context=config)
```